index.html

```diff
@@ -5,115 +5,151 @@
-                                <span>&#8674;</span>
-                            </div>
-                        </a>
-                    </div>
+                <article class="article">
+                    <a class="title" href="index.html">Complete Solution</a>
+                    <p class="underTitle">A tool anything and everything you can think</p>
+                    <img class="imgBig" src="img/tever.png" alt="">
+                    <p class="underImg">Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciun tdolore magnam aliquam quaerat voluptatem.</p>
+                    <a class="articleButton" href="index.html">
+                        <div class="buttonDiv">
+                            <span>Learn More</span>
+                            <span class="symbolRight">&#8674;</span>
+                        </div>
+                    </a>
                 </article>
-                <article class="verev">
-                    <div>
-                        <a class="vernagir" href="index.html">Uber Culture</a>
-                        <p class="under__a">Fresh. Modern and ready for future</p>
-                        <img class="img" src="img/tarer.png" alt="">
-                        <p class="under__img">Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem.</p>
-                        <a class="button" href="index.html">
-                            <div class="button__div">
-                                <span>Learn More</span>
-                                <span>&#8674;</span>
-                            </div>
-                        </a>
-                    </div>
+                <article class="article">
+                    <a class="title" href="index.html">Uber Culture</a>
+                    <p class="underTitle">Fresh. Modern and ready for future</p>
+                    <img class="imgBig" src="img/tarer.png" alt="">
+                    <p class="underImg">Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem.</p>
+                    <a class="articleButton" href="index.html">
+                        <div class="buttonDiv">
+                            <span>Learn More</span>
+                            <span class="symbolRight">&#8674;</span>
+                        </div>
+                    </a>
                 </article>
             </div>
-            <hr id="hr2" />
-            <div>
-                <article>
-                    <div>
-
+            <hr class="hr2" noshade color="#e4e4e4" width="929" />
+            <div class="sectionBottom">
+                <section class="section">
```

Comment:
remove section className, use tag selector instead.

Reply:
I did this for the reason on the page each hr has a different width

Reply:
I mentioned section classname not hr.